### PR TITLE
test: new tests for multi axis options (DHIS2-6672)

### PR DESCRIPTION
**Pull request by martinkrulltott**
Implements tests for [DHIS2-6672](https://jira.dhis2.org/browse/DHIS2-6672)

Tests the following options:

- Steps
- Decimals (can only be enabled but not evaluated)
- Auto generated axis titles for single and multi axis

image

### CODE REVIEW

File: cypress/elements/optionsModal/axes.js
Comment (jenniferarnesen) [inline cypress/elements/optionsModal/axes.js:22]: These 3 functions could be made into a single function `setAxisTitleTo(option)` or maybe `setAxisTitleOptionTo(option)`. In the test it would look like:

`setAxisTitleOptionTo('None')`

which one could argue is more readable than

`setAxisTitleOptionToNone()`

Just a thought. It's your call.

Comment (martinkrulltott) [reply cypress/elements/optionsModal/axes.js:22]: Good idea 👍

File: cypress/integration/options/axes.cy.js
Comment (jenniferarnesen) [inline cypress/integration/options/axes.cy.js:182]: There is some inconsistency of the phrasing of the test names in the test files: e.g. open vs opens, set vs sets. See line 182 vs 186, 190.

Not a big deal, but it would read better if it were consistent.

**Review by jenniferarnesen — APPROVED** [review]

File: cypress/integration/options/axes.cy.js
Comment (martinkrulltott) [reply cypress/integration/options/axes.cy.js:182]: Didn't see this comment until after I merged it (got trolled by GH... again). But I'll keep this in mind for sure!

**Pull request merged**
